README.md

```diff
@@ -57,6 +58,14 @@ These are satRdays organiser groups / locations that have already run a satRday
 ### US [0/3 slots filled]
 ### Rest of World [0/3 slots filled]
```

Comment:
Can you please also update the number of slots filled?

Reply:
I have done that. Thanks